Dockerfile.openvino

```diff
@@ -9,16 +9,7 @@ RUN apt-get update -y && \
         ffmpeg libsm6 libxext6 libgl1
 WORKDIR /workspace
 
-# copy requirements
```

Comment:
I think we copy files separately to avoid docker build cache invalidation.

Reply:
Is this a problem? The next step builds vllm, and that relies `sccache` anyway, which greatly speeds up the process.